frontend/src/components/Message/Message.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div>
+    <!-- Header -->
+    <div class="header py-lg-6">
+      <b-container>
+        <div class="header-body text-center mb-7">
+          <!-- eslint-disable-next-line @intlify/vue-i18n/no-dynamic-keys-->
+          <p class="h1 test-message-headline">{{ $t(headline) }}</p>
+          <!-- eslint-disable-next-line @intlify/vue-i18n/no-dynamic-keys-->
+          <p class="h4 test-message-subtitle">{{ $t(subtitle) }}</p>
+          <hr />
+          <b-button
+            v-if="buttonText"
+            class="test-message-button"
+            :to="linkTo + (code ? `/${code}` : '')"
+          >
+            <!-- eslint-disable-next-line @intlify/vue-i18n/no-dynamic-keys-->
+            {{ $t(buttonText) }}
+          </b-button>
+        </div>
+      </b-container>
+    </div>
+    <!-- Page content -->
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Message',
+  props: {
+    headline: { type: String, required: true },
+    subtitle: { type: String, required: true },
+    buttonText: { type: String, required: false, default: null },
+    linkTo: { type: String, required: false, default: null },
+    code: { type: String, required: false, default: null },
```

Comment:
Also consider to not include the code part in this component but have the component calling this one to provide a correct `linkTo` including the code already.

**I am unsure which way is better for this question**

Reply:
Now since I give the `linkTo` as general prop your suggestion is the best way.

I put it here before just to don't forget about it and care for a solution later if more gets clear.